apps/package.json

```diff
@@ -253,7 +253,7 @@
     "qrcode.react": "^0.8.0",
     "query-string": "4.1.0",
     "react-ace": "^9.2.1",
-    "react-csv": "^1.0.14",
+    "react-csv": "^2.0.3",
```

Comment:
Just checking that this change was intentional?

Reply:
yeah, the old version didn't have an `onClick` property. i documented that in the commit message but forgot to add it to the PR description.